lib/node_modules/@stdlib/stats/base/nanmskrange/docs/types/test.ts

```diff
@@ -26,7 +27,7 @@ import nanmskrange = require( './index' );
 	const x = new Float64Array( 10 );
 	const mask = new Uint8Array( 10 );
 
-	nanmskrange( x.length, x, 1, mask, 1 ); // $ExpectType number
+	nanmskrange( x.length, new AccessorArray( x ), 1, mask, 1 ); // $ExpectType number
```

Comment:
```suggestion
	nanmskrange( x.length, x, 1, mask, 1 ); // $ExpectType number
	nanmskrange( x.length, new AccessorArray( x ), 1, mask, 1 ); // $ExpectType number
```

You should keep the prior assertions.